packages/math-input/src/components/keypad/shared-keys.tsx

```diff
@@ -58,6 +60,46 @@ export default function SharedKeys(props: Props) {
             ? [3, 1]
             : [3, 0];
 
+    // We show a minimal sidebar for our mobile fraction view
+    if (fractionsOnly) {
```

Comment:
Wondering if we should do one of two things:
- move this to a separate component (if it's not too much trouble, I think that would be my instinct)
- OR rename this file. I could be off about this, but the name seems wrong. In Expression Keypad they're shared between tabs; but the fact that they're different between Expression / Fraction makes it feel like it's not shared.

Not going to lose sleep on this either way.